src/main/js/helpers/constants.js

```diff
@@ -149,7 +149,7 @@ export const AXES_ORIENTATION = {
 export default {
     PADDING: {
         top: 10,
-        bottom: 2.5,
+        bottom: 5,
```

Comment:
Does doing this fix the issue happening in the site right now (_The topmost values in the Y Axis are cut-off_)? 

![image](https://user-images.githubusercontent.com/4908886/71182583-ee9e7b80-223b-11ea-8bf2-f2e4c9c0a136.png)

Reply:
Yes, it fixes it.